### ISSUE TITLE
Real keyboard Example with sm_td

### ISSUE THREAD

**Issue by TheSkyHeart** [stasmarkin/sm_td#13]
Hi,
like everyone else here. I saw your Reddit post, and have been trying to use HMR but it isn't really usable, I'm not a fast typer 40-50 WPM.

after seeing your post I said this is it finally, but I'm really struggling with implementing it, I'm not really coder and most of the code we need to add I don't understand, so it would be really nice to have some real examples for keyboard like Corne, real examples with everything needed for a normal user to just add and use.

appreciate the effort in making this work, and Thank you.

**Comment by TheSkyHeart** [comment]
Thank you @azzamsa

**Comment by stasmarkin** [comment]
@TheSkyHeart, that's a good point, I was thinking about that too. I plan to make some "common" layout examples. I think Miroku's would be the first, and maybe I'll add some others.

@azzamsa also gave great advice, I would even improve it by searching for `include "sm_td.h"`.

**Comment by TheSkyHeart** [comment]
That would be really great @stasmarkin, Miroku's layout is also something I want to use but again with HMR in current state it isn't usable.

appreciate it, and will keep following the updates.

**Comment by azzamsa** [comment]
@TheSkyHeart I am using miryoku too, you could take inspirations from my keymap #14

**Comment by azzamsa** [comment]
> that's a good point, I was thinking about that too. I plan to make some "common" layout examples. I think Miroku's would be the first, and maybe I'll add some others.

Yes. Coming to this repo for the first time, I have no clue how to glue it together with my keymap.
I think having `example/` directory with super minimal keymap would do the job.

**Comment by stasmarkin** [comment]
moving this thread to new feature-ticket
https://github.com/stasmarkin/sm_td/issues/46
